crates/goose-cli/src/commands/mcp.rs

```diff
@@ -1,14 +1,15 @@
 use anyhow::Result;
-use goose_mcp::DeveloperRouter;
+use goose_mcp::{DeveloperRouter, JetBrainsRouter};
 use mcp_server::router::RouterService;
-use mcp_server::{ByteTransport, Server};
+use mcp_server::{BoundedService, ByteTransport, Server};
 use tokio::io::{stdin, stdout};
 
 pub async fn run_server(name: &str) -> Result<()> {
     tracing::info!("Starting MCP server");
 
-    let router = match name {
-        "developer" => Some(RouterService(DeveloperRouter::new())),
+    let router: Option<Box<dyn BoundedService>> = match name {
+        "developer" => Some(Box::new(RouterService(DeveloperRouter::new()))),
+        "jetbrains" => Some(Box::new(RouterService(JetBrainsRouter::new()))),
```

Comment:
should this be hard wired in or could be added to CLI?

Reply:
yes I added it hardcoded for now since we don't have dynamic loading working properly yet